regexes/device/mobiles.yml

```diff
@@ -7063,11 +7119,13 @@ Kaan:
 
 # Kazam
 Kazam:
-  regex: 'Kazam ([^;/)]+)( Build|\))|Trooper_X[0-9][0-9](?:[);/ ]|$)|Tornado 348'
+  regex: 'Kazam(?:[);/ ]|$)|Trooper_X[0-9][0-9](?:[);/ ]|$)|Tornado 348'
```

Comment:
Does that change make sense here? Don't think `Kazam` might be the end of the user agent. Maybe that should work:

```suggestion
  regex: 'Kazam(?:[^);/]+)(?:[);/ ]|$)|Trooper_X[0-9][0-9](?:[);/ ]|$)|Tornado 348'
```

Guess it should be change in the model regex as well

Reply:
I suggest replacing this rule with `Kazam(?:[^);/]+)(?:[);/ ]|$)|` the following  `Kazam[; _]|`  or `Kazam|`

Reply:
that's fine. you can also optimize the model regex a few lines below

Reply:
@sgiehl done